h/static/scripts/directive/annotation.js

```diff
@@ -734,6 +734,10 @@ function link(scope, elem, attrs, controllers) {
         if (threadFilter) {
           threadFilter.freeze(false);
         }
+        if (thread.parent) {
+          thread.parent.toggleCollapsed(false);
```

Comment:
This could use a brief one-line comment here to explain what is going on, eg. `// Ensure that the just-edited annotation remains visible`.